includes/admin/class-wc-admin-importers.php

```diff
@@ -159,7 +159,7 @@ public function post_importer_compatibility() {
 					foreach ( $post['terms'] as $term ) {
 						if ( strstr( $term['domain'], 'pa_' ) ) {
 							if ( ! taxonomy_exists( $term['domain'] ) ) {
-								$attribute_name = wc_sanitize_taxonomy_name( str_replace( 'pa_', '', $term['domain'] ) );
```

Comment:
Is this an err? Looks like the pa version was used here?

Reply:
The new function uses `wc_sanitize_taxonomy_name` before doing the str_replace so the only difference is the processing order change.

https://github.com/woocommerce/woocommerce/pull/22112/files#diff-dc3a1c9d68e161cfe6566b05971ec631R681